src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -204,6 +207,74 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
             }
         }
+
+        [RavenAction("/studio/whats-new", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetWhatsNewData()
+        {
+            if (ServerVersion.Build == ServerVersion.DevBuildNumber)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
```

Comment:
It's no a bad request. It should rather return `HttpStatusCode.NoContent`. Please check other similar endpoints or consult with @ml054

Reply:
Returning `NoContent` now